InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -153,13 +154,55 @@ void GenericHandler::showCommandPrompt(Visualization::Item* commandReceiver, QSt
 {
 	if (commandPrompt_ && commandPrompt_->commandReceiver() == commandReceiver)
 	{
-		commandPrompt_->showPrompt(initialCommandText);
+        commandPrompt_->showPrompt(initialCommandText);
```

Comment:
Ouch ouch ouch. I noticed that here and everywhere else you're using spaces instead of tabs for indentation. Please change your editor settings and fix all files. Refer to the envision style guides for the correct settings.